src/knexfile.js

```diff
@@ -73,7 +73,7 @@ module.exports = {
   },
   // Test with Oracle on Linux (e.g. on CircleCI)
   // Set environment variables
-  // NOTARY_DB_ENV=oracledci LD_LIBRARY_PATH='/opt/oracle/instantclient'
+  // LEDGER_DB_KNEX_ENV=oracledci LD_LIBRARY_PATH=/opt/oracle/instantclient
```

Comment:
What about not hardcoding the location of instantlclient either? Or at least setting this as default and letting me use an ENV variable to change it.